quiz/web/src/App.tsx

```diff
@@ -1,14 +1,31 @@
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { AuthConfig, AuthTokenProvider } from "modules/common/AuthToken";
+import ErrorBoundary from "modules/common/components/ErrorBoundary";
+import { BrowserRouter, createBrowserRouter, RouterProvider } from "react-router-dom";
+
+const router = createBrowserRouter([
```

Comment:
I think that will be better if we create separate file for the router. Might be big in the future.

Reply:
fixed